docs/.eleventy/layouts/main.njk

```diff
@@ -5,25 +5,34 @@ description: "Snowpack is the fast, bundle-free build tool for all kinds of fron
-      <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
-          <g id="mountain-solid" transform="translate(-1.000000, 0.000000)"fill-rule="nonzero">
-              <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
-          </g>
+  <svg height="64px" style="padding-left: 8px;" viewbox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
+      <g id="mountain-solid" transform="translate(-1.000000, 0.000000)" fill-rule="nonzero">
+        <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
       </g>
+    </g>
   </svg>
   <h1 class="header-snowpack">Snowpack</h1>
   <p class="header-snowpack-subtitle">
     The faster frontend build tool.
   </p>
+  <p class="header-snowpack-blurb">
+    Snowpack is an
+    <u>unbundled</u>
+    front-end development environment that starts in
+    <strong>50 milliseconds or less</strong>. Leveraging the power of native JavaScript ES6 modules, code changes show up instantly in the browser. When it's time to deploy, Snowpack collaborates with your favorite tools (including bundlers) to create a production-ready build.
+    <em>Use a bundler because you want to, not because you need to.</em>
+  </p>
+
   <div style="margin: 1.5rem auto; max-width: 424px;">
+
     <a href="#get-started" class="button">Get Started</a>
-  <!-- Place this tag where you want the button to render. -->
-  <div class="hidden-mobile" style="text-align: center;">
+    <!-- Place this tag where you want the button to render. -->
+    <div class="hidden-mobile" style="text-align: center;">
       <a class="github-button" href="https://github.com/pikapkg/snowpack" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star pikapkg/snowpack on GitHub">Star</a>
-  </div>
-  <!-- Place this tag in your head or just before your close body tag. -->
-  <script async defer src="https://buttons.github.io/buttons.js"></script>
+    </div>
+    <!-- Place this tag in your head or just before your close body tag. -->
+    <script async="async" defer="defer" src="https://buttons.github.io/buttons.js"></script>
```

Comment:
I think this should still be `<script async defer`? I don’t think it makes any difference—this probably works—but I think the way it was before is more standard/familiar to people.